src/data/labels/COCO.ts

```diff
@@ -0,0 +1,34 @@
+export type COCOSegmentation = number[][]
+
+export type COCOInfo = {
+    description: string;
+}
+
+export type COCOImage = {
+    id: number;
+    width: number;
+    height: number;
+    file_name: string;
+}
+
+export type COCOCategory = {
+    id: number;
+    name: string;
+}
+
+export type COCOAnnotation = {
+    id: number;
+    category_id: number;
+    iscrowd: number;
```

Comment:
ten snake_case w tych polach jest celowy? to jakaś konwencja COCO?
bo tutaj `iscrowd` zamiast `is_crowd` :P i trochę mi nie pasuje xD